docs/migration_guide.rst

```diff
@@ -12,6 +12,8 @@ already modify your applications to use new deployment flow.
 Breaking Changes
 ----------------
 
+- Removed deprecated `Account.sign_transaction`. Use new `Account.sign_invoke_transaction`.
+- Removed deprecated `InvokeFunction` as `call_contract` parameter. Use `Call` class instead.
```

Comment:
Do we plan to release it together with 0.10.1 basic changes?

Reply:
I would like to avoid adding breaking changes with the initial version supporting it.

Reply:
I wanted to include this with 0.10.1 post release. Better to batch these removals anyway since it's unlikely that users will upgrade cairo-lang mid starknet release cycle.

Reply:
Yes, post release is fine